docs/scanners/base.md

```diff
@@ -35,6 +35,20 @@ scanner_configs:
       Slack channel: #security
 ```
 
+#### Maximum Lifespan
+
+At times, scanners may perform scans for unacceptable lengths of time. To limit this behavior, you can define `scanner-timeout-ms` with the number of milliseconds you wish the scan to last before it times out.  
+
+Example with `YarnAudit` configuration:
+
+```yaml
+scanner_configs:
+  YarnAudit:
+    scanner-timeout-ms: 60000
+```
+
+This will limit YarnAudit scans to 1 minute (60,000 milliseconds) in execution time.
```

Comment:
Could you change miliseconds to seconds?
We pretty much just want to ensure the scanner doesn't run over x minutes or something, so miliseconds is too precise and not really needed.